src/azure-cli-testsdk/azure/cli/testsdk/decorators.py

```diff
@@ -1,24 +1,8 @@
-# --------------------------------------------------------------------------------------------
-# Copyright (c) Microsoft Corporation. All rights reserved.
-# Licensed under the MIT License. See License.txt in the project root for license information.
-# --------------------------------------------------------------------------------------------
-
-import os
 import unittest
-from azure.cli.core.profiles import supported_api_version
-from .const import ENV_LIVE_TEST
-
 
-def live_only():
-    return unittest.skipUnless(
-        os.environ.get(ENV_LIVE_TEST, False),
-        'This is a live only test. A live test will bypass all vcrpy components.')
+from azure_devtools.scenario_tests.decorators import live_only, record_only
```

Comment:
The point here was to keep compatibilty of people doing `from azure.cli.testsdk.decorators import live_only`. If this is not required, we can remove completely this line.